metricbeat/module/haproxy/haproxy.go

```diff
@@ -0,0 +1,243 @@
+package haproxy
+
+import (
+	"bytes"
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"github.com/gocarina/gocsv"
```

Comment:
Please separate the stdlib imports from the third-party imports. Most of our code follows this style if you would like to see some examples. Checkout the [goimports](http://goinbigdata.com/goimports-vs-gofmt/#goimports) tool.